de.metas.handlingunits.client/src/test/java/de/metas/handlingunits/client/terminal/editor/model/impl/MovementsAnyWarehouseTests.java

```diff
@@ -0,0 +1,115 @@
+package de.metas.handlingunits.client.terminal.editor.model.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.adempiere.model.InterfaceWrapperHelper;
+import org.adempiere.test.AdempiereTestHelper;
+import org.compiere.model.I_M_Locator;
+import org.compiere.model.I_M_Warehouse;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import de.metas.handlingunits.model.I_M_HU;
+
+/*
+ * #%L
+ * de.metas.handlingunits.client
+ * %%
+ * Copyright (C) 2017 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+public class MovementsAnyWarehouseTests
+{
+	@Before
+	public void init()
+	{
+		AdempiereTestHelper.get().init();
+	}
+
+	@Test
+	public void testGetWarehousesUnlessOfHUs()
+	{
+
+		final I_M_Warehouse warehouse1 = createWarehouse("Warehouse1");
+		final I_M_Warehouse warehouse2 = createWarehouse("Warehouse2");
+		final I_M_Warehouse warehouse3 = createWarehouse("Warehouse3");
+
+		final I_M_Locator locator1 = createLocator(warehouse1, "Locator1");
+		final I_M_Locator locator2 = createLocator(warehouse2, "Locator2");
+		final I_M_Locator locator3 = createLocator(warehouse3, "Locator3");
+
+		final I_M_HU hu1 = createHU(locator1);
+		final I_M_HU hu2 = createHU(locator2);
+		final I_M_HU hu3 = createHU(locator3);
+
+		List<I_M_HU> hus = new ArrayList<>();
+		hus.add(hu1);
+
+		final List<I_M_Warehouse> warehousesUnlessOfHUs1 = MovementsAnyWarehouseModel.getWarehousesUnlessOfHUs(hus);
+
+		Assert.assertFalse(warehousesUnlessOfHUs1.contains(warehouse1));
+		Assert.assertTrue(warehousesUnlessOfHUs1.contains(warehouse2));
```

Comment:
Please take a look at assertj..
you and e.g. write this one as
```java
assertThat(warehousesUnlessOfHUs1).contains(warehouse2);
```

Reply:
Thanks! 
I notices the assertThat in other tests but I didn't know how to import it. I found the import online and just added it manually: import static org.assertj.core.api.Assertions.*;